provisioning_templates/iPXE/ipxe_global_default.erb

```diff
@@ -0,0 +1,47 @@
+<%#
+kind: iPXE
+model: ProvisioningTemplate
+name: iPXE global default
+snippet: false
+-%>
+#!ipxe
+
+set menu-default <%= global_setting("default_pxe_item_global", "local") %>
+set menu-timeout 5000
+
+:start
+menu iPXE global boot menu
+item --key l local     Continue local boot
+item shell             Drop into iPXE shell
+item reboot            Reboot system
+item
+item --key d discovery Foreman Discovery
+choose --timeout ${menu-timeout} --default ${menu-default} selected || goto cancel
+set menu-timeout 0
+goto ${selected}
+
+:cancel
+echo Menu canceled, dropping to shell
+
+:shell
+echo Use the command 'exit' to return to menu
+shell
+set menu-timeout 0
+goto start
+
+:failed
+echo Boot failed, dropping to shell
+goto shell
+
+:reboot
+reboot
+
+:local
+exit
+
+:discovery
+dhcp
+kernel ${next-server}/boot/fdi-image/vmlinuz0 rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset proxy.url=<%= foreman_server_url %> proxy.type=foreman BOOTIF=01-${net0/mac}
+initrd ${next-server}/boot/fdi-image/initrd0.img
+boot || goto failed
+goto start
```

Comment:
Since this is an ERB we could generate multiple discovery menu items to be able to boot from eth0, eth1 up to 8. At least four would be very useful. But thanks :-)